docs/alerts.md

```diff
@@ -275,17 +288,14 @@ option task = {
 	every: 1m,
 }
 
-// Possibly an option like now?
-notify = (tables=<-) => tables |> to(bucket: "system")
+// Default value for the flux embedded in influxdb.
+option handleCheckResult = (tables=<-) => tables |> to(bucket: "system")
```

Comment:
I like the use of the option, I think it works well here. I agree the default should be to write to the bucket.

Reply:
One key aspect is I think it should only be the default in influxdb. I think the option should default to doing nothing in flux itself.

I hope I made that clear. I have some apprehensions with making any of the default behavior rely on code from influxdb. I hope that is understood and is ok.

Reply:
Yes, its clear, I don't have a strong opinion on the default inside vanilla Flux ATM. 

Implementation-wise I am not sure it is easy to have different defaults but we can figure that out later.